website/docs/reference/operators/evaluation-operators.md

```diff
@@ -0,0 +1,162 @@
+---
+sidebar_position: 5
+---
+
+# Evaluation query operators
+
+Evaluation query operators return data based on the evaluation of a specified expression.
+
+| Operator           | Description                                                                                   |
+| ------------------ | --------------------------------------------------------------------------------------------- |
+| [`$mod`](#mod)     | Matches documents where the value of a field divided by a divisor has the specified remainder |
```

Comment:
That's probably something very English-related, but why in this sentence do we say "a divisor" but "the specified remainder"? Both divisor and remainder are specified.